src/backend/libs/efiling-cso-starter/src/test/java/ca/bc/gov/open/jag/efilingcsostarter/csoAccountServiceImpl/UpdateClientTest.java

```diff
@@ -90,7 +90,22 @@ public void withNullClientIdReturnIllegalArgumentException() {
 
 
         Assertions.assertThrows(IllegalArgumentException.class, () -> {
-            sut.updateClient(null);
+            sut.updateClient(AccountDetails.builder()
+                    .internalClientNumber(INTERNAL_CLIENT_NUMBER1)
```

Comment:
this method is named
```withNullClientIdReturnIllegalArgumentException```
even though it is no longer `clientId` right?

Reply:
Client id is required for the update